go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/allegro/bigcache
+
+go 1.12
+
+require (
+	github.com/cespare/xxhash v1.1.0 // indirect
+	github.com/coocood/freecache v1.1.0
```

Comment:
Sounds like a good task for the future: remove freecache dependency.

Reply:
It's a dependency of our benchmarking and comparison tests, not a direct dependency of bigcache itself.

Reply:
Anyway that's useless 😉